src/main/java/eu/dissco/backend/repository/MasJobRecordRepository.java

```diff
@@ -123,6 +123,9 @@ private Record5<String, String, String, Instant, String> mjrToRecord(MasJobRecor
 
   private MasJobRecordFull recordToMasJobRecord(Record dbRecord) {
     try {
+      var dataNode = dbRecord.get(MAS_JOB_RECORD.ANNOTATIONS) != null ?
+          mapper.readValue(dbRecord.get(MAS_JOB_RECORD.ANNOTATIONS).data(), JsonNode.class) :
+          null;
       return new MasJobRecordFull(
```

Comment:
This function takes the contents pulled from the database into a Java Object. 

`mapper.readValue(dbRecord.get(MAS_JOB_RECORD.ANNOTATIONS).data(), JsonNode.class)` -> takes the value stored in the ANNOTATIONS column in the MAS_JOB_RECORD table and maps it to a JSON Object (which is part of the Java Object MasJobRecordFull). 

Before, if the value of ANNOTATIONS col was null, a null pointer exception (NPE) would be thrown. If this column is null, we automatically set the MasJobRecordField, without triggering the NPE.